src/docs/sdks/react/index.mdx

```diff
@@ -0,0 +1,179 @@
+
+**Important:** Note your DSN. The _DSN_ (Data Source Name) tells the SDK where to send events. If you forget it, view _Settings -> Projects -> Client Keys (DSN)_ in the Sentry web UI.
+
+### Add React Error Boundary
+
+If you're using React 16 or above, Error Boundaries are an essential tool for defining your application's behavior in the face of errors. The `@sentry/react` package exposes an error boundary component that automatically sends JavaScript errors from inside a React component tree to Sentry. You can use the error boundary just like a regular React component.
+
+```jsx
+import React from "react";
+import * as Sentry from "@sentry/react";
+
+<Sentry.ErrorBoundary fallback={"An error has occured"}>
+  <Example />
+</Sentry.ErrorBoundary>;
+```
+
+Learn how to further configure and use the Sentry Error Boundary in [React Error Boundary](./api/errorboundary/)
+
+## Verify Setup
+
+Verify your setup by intentionally sending an error that breaks your application. For example, in JavaScript, calling an undefined function throws an exception:
+
+```JavaScript
+myUndefinedFunction();
+```
+
+Resolve the created error by logging in to Sentry, then opening your project. On the **Issue Details** page, scroll down to the Exception stack trace and view the error.
+
+## Capture Errors
+
+A few key terms to know as we proceed:
+
+- An _event_ is one instance of sending data to Sentry. Generally, this data is an error or exception.
+- An _issue_ is a grouping of similar events.
+- The reporting of an event is called _capturing_.
+  When an event is captured, it’s sent to Sentry.
+
+By including and configuring Sentry, our JavaScript SDK automatically attaches global handlers to _capture_ uncaught exceptions and unhandled promise rejections, as described in the official ECMAScript 6 standard. You can disable this default behavior by changing the `onunhandledrejection` option to `false` in your GlobalHandlers integration and manually hook into each event handler, then call `Sentry.captureException` or `Sentry.captureMessage` directly.
+
+You may also need to manage your configuration if you are using a third-party library to implement promises. Also, keep in mind that browsers may take security measures when serving script files from different origins that can block error reporting.
+
+Learn more about how to manually capture errors or enable message capture with Sentry's JavaScript SDK in [Capture Errors](./errors/capture).
+
+### Automatically Enrich Error Data
+
+Events sent by the JavaScript SDK to Sentry are enriched with data that helps identify the source of the event. Much of this data is sent automatically - including the error context and environment - as well as the trail of events that led up to the event, which we call breadcrumbs. You don't need to configure these, though you may modify them.
+
+Learn more about the data sent to with events in [Enrich Event Data](./errors//).
```

Comment:
```suggestion
Learn more about the data sent to with events in [Enrich Event Data](./errors/).
```

Reply:
Also, "data sent to with events" -- something doesn't sound right there, typo? Maybe:

```suggestion
Learn more about the data sent with events in [Enrich Event Data](./errors/).
```